main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/joho/godotenv"
+	"github.com/vanduc1102/ondemand-go-bootcamp/src/controllers"
```

Comment:
Another good practice even though the go extension in VS Code tries to separate them is to separate core packages, then packages from your project, and finally third-party ones, this makes it easier for others to understand the project.
```suggestion
    "log"
	"net/http"
	"time"
	
	"github.com/vanduc1102/ondemand-go-bootcamp/src/controllers"
	
	"github.com/gin-gonic/gin"
	"github.com/joho/godotenv"
```